packages/compass-import-export/src/components/export-select-output/export-select-output.tsx

```diff
@@ -1,14 +1,12 @@
 import { SelectFileType } from '../select-file-type';
-import { IconTextButton } from 'hadron-react-buttons';
 import toNS from 'mongodb-ns';
-import fileSaveDialog from '../../utils/file-save-dialog';
 import ProgressBar from '../progress-bar';
 import { FILETYPE } from '../../constants/export-step';
+// @ts-expect-error No types for .less?
```

Comment:
You can add something like this to typings.d.ts for this package to be handled semi-correctly:

https://github.com/mongodb-js/compass/blob/ea4f317e964c0504e54c01d83c385280a9135bba/packages/compass-aggregations/src/typings.d.ts#L8-L11

Reply:
Thank you! Done :)